app/views/sessions/home.html.erb

```diff
@@ -1,24 +1,33 @@
 <style>
     body {
         flex: 1 1 auto;
+        text-align: center;
     }
 
     div.aRoute {
+        position:relative; 
+        left:80px; 
+        top:80px;
         background-color: #eeeeee;
         height: 400px;
         width: 370px;
         padding: 100px;
         border-radius: 10px;
+        opacity: 0.9;
     }
 
     li {
         font-size: 1.05em;
         list-style: none;
         padding-right: 20px;
     }
+
+    button {
+        color: black;
+    }
 </style>
 </br>
-<p> book to your liking today </p>
```

Comment:
Had to rid to try and fix some position issues I was having. Decided not to add it back